servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/HttpTransportObserverTest.java

```diff
@@ -250,7 +250,7 @@ void testRequestResponse(StreamingHttpRequest request, HttpResponseStatus expect
     @EnumSource(HttpProtocol.class)
     void serverFailsResponsePayloadBodyBeforeRead(HttpProtocol httpProtocol) throws Exception {
         setUp(httpProtocol);
-        testServerFailsResponsePayloadBody(SVC_ERROR_BEFORE_READ, false);
+        testServerFailsResponsePayloadBody(SVC_ERROR_BEFORE_READ, true);
```

Comment:
@idelpivnitskiy - inbound events are queued now so the server still reads complete even though write errors out. wdyt?

Reply:
IIUC this happens only when an incoming msg is already read from the socket and we fail a write later. If we can still complete reading this is a nice improvement!